packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -404,15 +405,11 @@ export class MuteState {
 
     // todo: I'm seeing "you can now unmute yourself " popup  when this happens - but same thing happens on web.w.c so we can ignore for now
     this.state.server.remoteMute = false;
+    this.state.client.localMute = false;
```

Comment:
this doesn't look right, I think it's a merge conflict with Bryce's changes for hardware mute that's resolved incorrectly, we can't just set localMute to false as there may still be hardware mute which we cannot override